agent/taskresource/credentialspec/credentialspec_windows.go

```diff
@@ -348,6 +410,24 @@ func (cs *CredentialSpecResource) clearCredentialSpec() {
 	}
 }
 
+func (cs *CredentialSpecResource) deleteTaskExecutionCredentialsRegKeys() {
+	cs.lock.Lock()
+	defer cs.lock.Unlock()
+
+	pathToTaskCredentialsRegistryKey := servicesRegistryKeyRoot + "\\" + taskCredentialsRegistryKey
+	k, err := registry.OpenKey(registry.LOCAL_MACHINE, pathToTaskCredentialsRegistryKey, registry.ALL_ACCESS)
+	if err != nil {
+		seelog.Errorf("Error opening %s key to delete: %s", pathToTaskCredentialsRegistryKey, err)
+	}
+	defer k.Close()
+
+	err = registry.DeleteKey(k, cs.taskARN)
+	if err != nil {
+		seelog.Errorf("Error deleting %s key: %s", pathToTaskCredentialsRegistryKey+"\\"+cs.taskARN, err)
```

Comment:
Same here. We dont want to log on 428 if it didnt succeed so needs a return

Reply:
done